connect-inject/container_init_test.go

```diff
@@ -589,7 +591,29 @@ services {
 		t.Run(tt.Name, func(t *testing.T) {
 			require := require.New(t)
 
-			var h Handler
+			// Create a Consul server/client and proxy-defaults config because
```

Comment:
These tests didn't change but now require consulClient to be instantiated because they test datacenter upstreams.